README.md

```diff
@@ -11,9 +11,81 @@ Federalist is a unified interface for publishing static government websites. It
 
 ## Getting started
 
-To run the server, you'll need [Node.js](https://nodejs.org/download/). You'll also need [nvm](https://github.com/creationix/nvm).
+### Dependencies / Tooling
+Before you start, ensure you have the following installed:
+
+- [node](https://nodejs.org/en/download/package-manager/#osx)
+- [nvm](https://github.com/creationix/nvm#installation) or [n](https://github.com/tj/n#installation)
+- [yarn](https://yarnpkg.com/)
+- [Postgres](https://www.postgresql.org/)
+- [Docker](https://docs.docker.com/engine/installation/mac/)
+- [Cloud Foundry CLI](https://docs.cloudfoundry.org/cf-cli/install-go-cli.html)
+- [AWS CLI](https://aws.amazon.com/cli/)
+
+
+### Running locally (development)
+
+1. Download or clone the `18F/federalist` repository from Github.
+1. Run `nvm use` to ensure you are using the correct version of node
+1. Run `yarn` from the root (the directory that houses the projects files on your computer) of the repository to load modules and install Jekyll dependencies
+1. Make a copy of `config/local.sample.js` and name it `local.js` and place it in the `config` folder. 
+This will be the file that holds your S3 and SQS configurations.
+1. [Register a new OAuth application on GitHub](https://github.com/settings/applications/new). You'll want to use `http://localhost:1337/auth` as the "Authorization callback url". Once you have created the application, you'll see a `Client ID` and `Client Secret`. Add these values to `config/local.js`.
+    ```
```

Comment:
in GitHub Markdown, you can "hint" the syntax of the codeblock to get specialized syntax highlighting. You do this basically by putting the file extension of what the code would be if it was in a file after the three backticks

For these samples of json, you'd do "```json"